<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="wid_top" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-1</y>
      <width>831</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wid_top{
	background-color:rgb(255, 87, 87);
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>15</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{
	color:rgb(249, 249, 249);
	background-color: rgb(255,87,87)
}</string>
     </property>
     <property name="text">
      <string>PARKING MANAGEMENT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wid_in" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

#wid_in{
background-color: rgb(255, 255, 255);
    border: 5px solid rgb(255, 87, 87);
}

</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">label_4{
	border: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Latest In</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>271</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	border: 5px solid rgb(255, 87,87);
}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>115</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UID Card</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TIME IN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="uid_in">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time_in_in">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wid_out" native="true">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>111</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#wid_out{
background-color: rgb(255, 255, 255);
    border: 5px solid rgb(255, 87,87);
}

</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft New Tai Lue</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Latest Out</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>271</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	border: 5px solid rgb(255, 87,87);
}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>131</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>137</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>141</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>137</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>131</width>
       <height>49</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QColumnView" name="columnView_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>141</width>
       <height>49</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{border: 5px solid rgb(255, 87, 87);}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UID Card</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TIME IN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TIME OUT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="uid_out">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>49</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time_out_in">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time_out_out">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>141</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>661</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#tabWidget{
    background-color: rgb(255, 87, 87);
 border: 3px solid rgb(255, 87, 87);
}
</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <widget class="QTextBrowser" name="log">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Parking</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
	color:rgb(255, 255, 255)
}</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SLOT 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SLOT 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="slot_1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>57</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="slot_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>57</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <widget class="QTextBrowser" name="his">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
